### ISSUE TITLE
run grassroots fundraising campaign

### ISSUE THREAD

**Issue by lianilychee** [open-austin/org#386]
- [ ] #416
- [ ] #417

**Comment by lianilychee** [comment]
closing this ticket, created two more granualr tix to cover this work

**Comment by jcklpe** [comment]
We might at some point adopt a sort of "Epic" process for collating more granular tasks under. But that can be dealt with later. @lianilychee

**Comment by lianilychee** [comment]
Yeah let's deal with that later. To be, "later" means when we have other people actually scrubbing in - otherwise we're just creating additional project/ticketing overhead
